commands/issue.go

```diff
@@ -428,3 +437,55 @@ text is the title and the rest is the description.`, project))
 
 	messageBuilder.Cleanup()
 }
+
+func listLabels(cmd *Command, args *Args) {
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.MainProject()
+	utils.Check(err)
+
+	gh := github.NewClient(project.Host)
+
+	if args.Noop {
+		ui.Printf("Would request list of labels for %s\n", project)
+		return
+	}
+
+	labels, err := gh.FetchLabels(project)
+	utils.Check(err)
+
+	for _, label := range labels {
+		// TODO add a colorize flag for the labels command
+		ui.Printf(formatLabel(label, true))
+	}
+
+	args.NoForward()
+}
+
+func formatLabel(label github.IssueLabel, colorize bool) string {
+	format := "%l%n"
+	if colorize {
+		format = "%c%n"
+	}
+
+	color, err := utils.NewColor(label.Color)
+	if err != nil {
+		utils.Check(err)
+	}
+
+	placeholders := map[string]string{
+		"l": label.Name,
+		"c": fmt.Sprintf("\033[38;5;%d;48;2;%d;%d;%dm %s \033[m",
+			getSuitableTextColor(color), color.Red, color.Green, color.Blue, label.Name),
+	}
+
+	return ui.Expand(format, placeholders, colorize)
```

Comment:
I think `ui.Expand` with placeholders might have been an overkill here. If the user doesn't have the option to pass the `--format` string (like for `hub issue` or `hub pr list`), then we could get away with simply:

```go
if colorize {
  return fmt.Sprintf("output with color", ...)
} else {
  return fmt.Sprintf("%s\n", label.Name)
}
```